src/solvers/flattening/arrays.cpp

```diff
@@ -8,6 +8,9 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "arrays.h"
 
+#include <cassert>
+#include <iostream>
+
```

Comment:
Don't include these. All outputs should be in appropriate mstreamt and assertions should be replaced by invariants and such.